docs/user-guide/index.rst

```diff
@@ -30,8 +30,8 @@
 :ref:`Model Alignment by SteerLM 2.0 Method <model-aligner-steerlm2>`
    SteerLM 2.0 is an extenstion to SteerLM method that introduces an iterative training procedure to explicitly enforce the generated responses to follow the desired attribute distribution.
```

Comment:
fix spelling.

SteerLM 2.0 is an extension to SteerLM method that introduces an iterative training procedure to explicitly enforce the generated responses to follow the desired attribute distribution.

Reply:
```suggestion
   SteerLM 2.0 is an extension to SteerLM method that introduces an iterative training procedure to explicitly enforce the generated responses to follow the desired attribute distribution.
```